<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpWindow</class>
 <widget class="QWidget" name="SignUpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="Text1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>enter username :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enteredPass">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>enter password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="enteredUser">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="reEnterdPass">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>re-enter password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
